examples/evidence-based-go.yml

```
@@ -0,0 +1,142 @@
+    path: 'internal/domain/**'
+    dependsOn: []  # Domain has no dependencies
+
+  - name: 'application'
+    path: 'internal/application/**'
+    dependsOn: ['domain']
+
+  - name: 'infrastructure'
+    path: 'internal/infrastructure/**'
+    dependsOn: ['domain', 'application']
+
+rules:
```

YAMLlint (1.37.1)

[error] duplication of key "rules" in mapping

(key-duplicates)